ui/src/app/pages/analysis/notebook-resource-card.tsx

```diff
@@ -0,0 +1,172 @@
+import {RenameModal} from 'app/components/rename-modal';
+import {Action, ResourceCardTemplate} from 'app/components/resource-card-template';
+import {withConfirmDeleteModal, WithConfirmDeleteModalProps} from 'app/components/with-confirm-delete-modal';
+import {withErrorModal, WithErrorModalProps} from 'app/components/with-error-modal';
+import {withSpinnerOverlay, WithSpinnerOverlayProps} from 'app/components/with-spinner-overlay';
+import {CopyNotebookModal} from 'app/pages/analysis/copy-notebook-modal';
+import {dropNotebookFileSuffix} from 'app/pages/analysis/util';
+import {workspacesApi} from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import {formatRecentResourceDisplayDate} from 'app/utils';
+import {RecentResource} from 'generated/fetch';
+import * as fp from 'lodash';
+import * as React from 'react';
+
+interface Props extends WithConfirmDeleteModalProps, WithErrorModalProps, WithSpinnerOverlayProps {
+  resource: RecentResource;
+  onUpdate: Function;
+}
+
+interface State {
+  showCopyNotebookModal: boolean;
+  showRenameModal: boolean;
+}
+
+export const NotebookResourceCard = fp.flow(
+  withErrorModal(),
+  withConfirmDeleteModal(),
+  withSpinnerOverlay(),
+)(class extends React.Component<Props, State> {
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      showCopyNotebookModal: false,
+      showRenameModal: false
+    };
+  }
+
+  get resourceType(): string {
+    return 'Notebook';
+  }
+
+  get displayName(): string {
+    return dropNotebookFileSuffix(this.props.resource.notebook.name);
+  }
+
+  get resourceUrl(): string {
+    const {workspaceNamespace, workspaceFirecloudName, notebook} =
+      this.props.resource;
+
+    return `/workspaces/${workspaceNamespace}/${workspaceFirecloudName}
```

Comment:
Newlines in backticks do not work as you might expect - this is injecting escaped spaces into the URL.